packages/i18n/tools/pot-to-php.js

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env node
```

Comment:
Did you run this file through some formatter? It's got all sorts of styling issues which aren't present in the same file from the Gutenberg repository (spaces instead of tabs, double quotes instead of single quotes).

Reply:
Probably my default prettier setup :P I'll fix it